Library/Formula/git.rb

```diff
@@ -137,6 +138,12 @@ def install
     # To avoid this feature hooking into the system OpenSSL, remove it.
     # If you need it, install git --with-brewed-openssl.
     rm "#{libexec}/git-core/git-imap-send" if build.without? "brewed-openssl"
+
+    if build.dsym?
+      install_dsym
+      system "make", "clean"
```

Comment:
The `make clean`s shouldn't be necessary; the build directory is cleaned after the build finishes anyway.